packages/server/lib/browsers/electron.ts

```diff
@@ -384,6 +384,15 @@ export = {
     // adds a header to the request to mark it as a request for the AUT frame
     // itself, so the proxy can utilize that for injection purposes
     win.webContents.session.webRequest.onBeforeSendHeaders((details, cb) => {
+      const defaultHeaders = {
```

Comment:
Same as with the extension, I think this should be called  `requestModifications` or something. Electron calls it [beforeSendResponse](https://www.electronjs.org/docs/latest/api/web-request#webrequestonbeforesendheadersfilter-listener), which makes sense, but again the request/response juxtaposition seems confusing.

Reply:
addressed in a2ad929192